components/tools/OmeroWeb/omeroweb/api/views.py

```diff
@@ -479,18 +573,23 @@ def get_opts(self, request, **kwargs):
         # at /plates/:plate_id/wells/ we have 'plate_id' in kwargs
         if 'plate_id' in kwargs:
             opts['plate'] = long(kwargs['plate_id'])
+        elif 'plateacquisition_id' in kwargs:
```

Comment:
I don't think there's a case for that, since plateacquisition ID would also define plate ID.